src/main/webapp/WEB-INF/view/profile-pages.jsp

```diff
@@ -20,11 +35,53 @@
 
   <!--TODO(asma526):
 
-  Add an about me box that can only be edited by the user
-  Add a box that displays all messages sent out by the user 
+  Add profile pic option
 
   -->
+  <div id="container">
+
+    <% if(request.getAttribute("error") != null){ %>
+        <h2 style="color:red"><%= request.getAttribute("error") %></h2>
+    <% } %>
```

Comment:
If there is an error, do we still want to show everything below?

Reply:
I dont think it does because when I change != null to == null and run the code it only displays null in red.

Reply:
Okay i think i got it now and no it is not intended